bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -79,7 +79,7 @@ export class Wallet {
      * Destroy the Wallet and drop its database connection.
      */
     async destroy(): Promise<void> {
-        await this.methodHandler.destroy();
+        return this.methodHandler.destroy();
```

Comment:
For my own understanding, can you explain?

Reply:
I mean, I'm assuming this is very similar to rust and this would have returned void instead of the future?

Reply:
yea its confused me as well a bit, and is essentially the same. This explanation helped me:
https://jakearchibald.com/2017/await-vs-return-vs-return-await/